generator/generator.yml

```diff
@@ -361,3 +361,19 @@ modules:
       - picoMobileMIB
       - picoIPv4MIB
       - picoIPv6MIB
+
+  raritan:
+    walk:
+      - sysUpTime
+      - interfaces
+      - 1.3.6.1.4.1.13742.4.1.3.1.5    # unitCpuTemp
+      - 1.3.6.1.4.1.13742.4.1.20.2.1.7 # inletCurrent
+      - 1.3.6.1.4.1.13742.4.1.20.2.1.8 # inletVoltage
+      - 1.3.6.1.4.1.13742.4.1.20.2.1.9 # inletActivePower
+      - 1.3.6.1.4.1.13742.4.1.2.2.1.3  # outletOperationalState
+      - 1.3.6.1.4.1.13742.4.1.2.2.1.4  # outletCurrent
+      - 1.3.6.1.4.1.13742.4.1.2.2.1.5  # outletMaxCurrent
+      - 1.3.6.1.4.1.13742.4.1.2.2.1.6  # outletVoltage
+      - 1.3.6.1.4.1.13742.4.1.2.2.1.7  # outletActivePower
+    auth:
+      community: raritan_private
```

Comment:
Is this the default for this hardware?

Reply:
Good point, I could probably move this to "raritan_public" since I'm not doing write access.  Here's where the community string is documented, and it matches the factory reset defaults:

http://support.raritan.com/PX/version-1.5.7/quick-setup-guides/qsg-dpx-0n-v1-5-5-e.pdf

Reply:
Yes, `raritan_public` would be better.